commercial/app/views/subscriberNumberPage.scala.html

```diff
@@ -0,0 +1,25 @@
+@(subscriberNumberPage: Page)(implicit request: RequestHeader)
+
+@import model.Page
+
+    @main(subscriberNumberPage) { } {
+        <div class="l-side-margins">
+            <div class="live-events-survey-wrapper" data-link-name="subscriber number page">
+                <div class="live-events-survey-wrapper-inner">
+                    <h1>Please enter your current subscription number below:</h1>
+                    <div class="subscriber-number">
+                        <form class="subscriber-number-form js-subscriber-number-form">
+                            <input type="text" name="number" id="number" class="input-number"/>
+                            <button type="submit" name="submit" class="button--primary">Submit</button>
+                        </form>
+                        <p class="subscriber-number-correct standfirst js-subscriber-number-correct u-h">
```

Comment:
I think it's better if using `hidden` instead of `u-h` in terms of accessibility. The former won't be read by voice-based UA-s.